### PR TITLE
Fix plugins (names, constants, FQCNs in examples)

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Various fixes for plugins which we apparently forgot to do:
- cobbler inventory plugin's `name` needs FQCN (5015d180beaeda3af3edf455503aa88a8c8ae7ff)
- oc connection plugin needs FQCN in `transport` (0ce20e22f984d9e6982e1d6fcf2ede239cb31aeb)
- inventory plugins need FQCN in examples, and allow FQCN in `plugin` option (23d9102ddb459ddebbe90186df4dfb41f76872ee)
- lookup plugins need FQCN in examples (d55a3f7295b0d40c79b086e6e9c3d33c18e9b632)
- callback plugins need FQCN in examples (cd7427e93bbfdf349eda3aef6d93e2282f3f3266)

##### ISSUE TYPE
- Bugfix Pull Request
- Docs Pull Request

##### COMPONENT NAME
plugins

### CODE REVIEW

File: plugins/inventory/cobbler.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
       plugin:
         description: The name of this plugin, it should always be set to C(cobbler) for this plugin to recognize it as it's own.
```
Comment (Andersson007) [inline plugins/inventory/cobbler.py:20]:
```suggestion
        description: The name of this plugin, it should always be set to C(cobbler) for this plugin to recognize it as it's own.
```
shouldn't it be changed too? and in the other files here

File: plugins/inventory/docker_machine.py [hunk 2]
```diff
@@ -24,7 +24,7 @@
         plugin:
             description: token that ensures this is a source file for the C(docker_machine) plugin.
```
Comment (Andersson007) [inline plugins/inventory/docker_machine.py:25]:
```suggestion
            description: token that ensures this is a source file for the C(docker_machine) plugin.
```

Comment (Andersson007) [reply plugins/inventory/docker_machine.py:25]: @felixfontein ^

**Comment by felixfontein** [comment]
@Andersson007 @gundalow thanks a lot for reviewing this!

**Pull request merged**